asistente_ladm_col/logic/quality_rules/qr_building_duplicate_records.py

```diff
@@ -0,0 +1,97 @@
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames, LayerConfig
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4016,
+                                                           QRE_IGACR4016E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRDuplicateBuildingRecords(AbstractLogicQualityRule):
+    """
+    Check that building don't have duplicate records
+    """
+    _ERROR_01 = QRE_IGACR4016E01  # Building with duplicate records
+
+    def __init__(self):
+        AbstractLogicQualityRule.__init__(self)
+
+        self._id = QR_IGACR4016
+        self._name = "Revisar que la tabla construccion no tenga registros repetidos"
+        self._tags = ["igac", "instituto geográfico agustín codazzi", "lógica", "negocio", "construccion", "registros repetidos"]
+        self._models = [LADMNames.SURVEY_MODEL_KEY]
+
+        self._errors = {self._ERROR_01: "Construccion no debe tener registros repetidos"}
+
+        # Optional. Only useful for display purposes.
+        self._field_mapping = dict()  # E.g., {'id_objetos': 'ids_punto_lindero', 'valores': 'conteo'}
+
+    def layers_config(self, names):
+        return {QUALITY_RULE_LADM_COL_LAYERS: [names.LC_BUILDING_T]}
+
+    def _validate(self, db, db_qr, layer_dict, tolerance, **kwargs):
+        self.progress_changed.emit(5)
+        ladm_queries = self._get_ladm_queries(db.engine)
+
+        pre_res, pre_obj = self._check_prerrequisite_layers(layer_dict)
+        if not pre_res:
+            return pre_obj
+
+        error_state = None
+        
+        fields = LayerConfig.get_logic_consistency_tables(db.names).get(db.names.LC_BUILDING_T)
+        # Check building with duplicate records
+        res, records = ladm_queries.get_duplicate_records_in_table(db,db.names.LC_BUILDING_T,fields)
+        count = len(records)
+
+        self.progress_changed.emit(50)
+
+        if res:
+            error_state = LADMData().get_domain_code_from_value(db_qr, db_qr.names.ERR_ERROR_STATE_D,
+                                                                LADMNames.ERR_ERROR_STATE_D_ERROR_V)
+            errors = {'geometries': list(), 'data': list()}
+            for record in records:
+                error_data = [  # [obj_uuids, rel_obj_uuids, values, details, state]
+                    record['duplicate_uuids'].split(','),
+                    None,
+                    None,
+                    'Construcción repetido {} veces'.format(record['duplicate_total']),
```

Comment:
change for: 'Construcción repetida {} veces'

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/547/commits/955b7fa942b033b1d5edf8d4f3d6ce7a590ca9da